client/state/purchases/reducer.js

```diff
@@ -112,7 +112,7 @@ function updatePurchases( existingPurchases, action ) {
 
 const assignError = ( state, action ) => ( { ...state, error: action.error } );
 
-export const items = createReducer( initialState, {
+export default createReducer( initialState, {
```

Comment:
Would you mind explaining this change? Nesting this reducer under `items` gives us flexibility if we ever need to add additional reducers, and is a somewhat common pattern in `client/state/`. I don't see the utility in changing it, but I'm sure it exists. :)